sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -1,5759 +1,5763 @@
-      },
-      "validateOn": "change",
-      "validate": {
-        "required": false,
-        "custom": "",
-        "customPrivate": false,
-        "strictDateValidation": false,
-        "multiple": false,
-        "unique": false
-      },
-      "conditional": {
-        "show": null,
-        "when": null,
-        "eq": ""
-      },
-      "allowCalculateOverride": false,
-      "showCharCount": false,
-      "showWordCount": false,
-      "allowMultipleMasks": false,
-      "addons": [],
-      "inputType": "hidden",
-      "id": "etv9r3"
-    },
-    {
-      "label": "applicationStatus",
-      "labelWidth": "",
-      "labelMargin": "",
-      "customClass": "",
-      "modalEdit": false,
-      "defaultValue": null,
-      "persistent": "client-only",
-      "protected": false,
-      "dbIndex": false,
-      "encrypted": false,
-      "redrawOn": "",
-      "customDefaultValue": "",
-      "calculateValue": "",
-      "calculateServer": false,
-      "key": "applicationStatus",
-      "tags": [],
-      "properties": {},
-      "logic": [],
-      "attributes": {},
-      "overlay": {
-        "style": "",
-        "page": "",
-        "left": "",
-        "top": "",
-        "width": "",
-        "height": ""
-      },
-      "type": "hidden",
-      "input": true,
-      "tableView": false,
-      "placeholder": "",
-      "prefix": "",
-      "suffix": "",
-      "multiple": false,
-      "unique": false,
-      "hidden": false,
-      "clearOnHide": true,
-      "refreshOn": "",
-      "dataGridLabel": false,
-      "labelPosition": "top",
-      "description": "",
-      "errorLabel": "",
-      "tooltip": "",
-      "hideLabel": false,
-      "tabindex": "",
-      "disabled": false,
-      "autofocus": false,
-      "widget": {
-        "type": "input"
-      },
-      "validateOn": "change",
-      "validate": {
-        "required": false,
-        "custom": "",
-        "customPrivate": false,
-        "strictDateValidation": false,
-        "multiple": false,
-        "unique": false
-      },
-      "conditional": {
-        "show": null,
-        "when": null,
-        "eq": ""
-      },
-      "allowCalculateOverride": false,
-      "showCharCount": false,
-      "showWordCount": false,
-      "allowMultipleMasks": false,
-      "addons": [],
-      "inputType": "hidden",
-      "id": "ejidpjh"
-    },
-    {
-      "input": true,
-      "tableView": true,
-      "key": "studyPeriodMinDays",
-      "label": "Study period minimum days",
-      "protected": false,
-      "unique": false,
-      "persistent": true,
-      "type": "hidden",
-      "tags": [],
-      "conditional": {
-        "show": "",
-        "when": null,
-        "eq": ""
-      },
-      "properties": {},
-      "lockKey": true,
-      "customDefaultValue": "",
-      "calculateValue": "value = data.offeringIntensity === 'Full Time' ? 84 : 42;"
-    },
-    {
-      "input": true,
-      "tableView": true,
-      "key": "studyPeriodMaxDays",

```

Comment:
Was there a requirement to remove the max days validation ? and hence the `studyPeriodMaxDays` hidden variable?